account-decoder/src/parse_token.rs

```diff
@@ -15,16 +15,31 @@ use {
 
 // A helper function to convert spl_token::id() as spl_sdk::pubkey::Pubkey to
 // solana_sdk::pubkey::Pubkey
-pub fn spl_token_id() -> Pubkey {
+fn spl_token_id() -> Pubkey {
     Pubkey::new_from_array(spl_token::id().to_bytes())
 }
 
+// Returns all known SPL Token program ids
+pub fn spl_token_ids() -> Vec<Pubkey> {
+    vec![spl_token_id()]
+}
+
+// Check if the provided program id as a known SPL Token program id
+pub fn is_known_spl_token_id(program_id: &Pubkey) -> bool {
+    *program_id == spl_token_id()
```

Comment:
...and also this list.  

Perhaps a premature optimization that outweighs the burden of having two places to update? `is_known_spl_token_id()` gets called frequently, so I didn't feel great about having it call `spl_token_ids()` to build and search the vec for a match.

Reply:
Seems fine to me